config/job.js

```diff
@@ -163,6 +163,12 @@ const SCHEMA_BLOCKEDBY_VALUE = Joi.alternatives().try(
 );
 const SCHEMA_BLOCKEDBY = Joi.alternatives().try(Joi.array().items(SCHEMA_BLOCKEDBY_VALUE), SCHEMA_BLOCKEDBY_VALUE);
 const SCHEMA_FREEZEWINDOWS = Joi.alternatives().try(Joi.array().items(SCHEMA_CRON_EXPRESSION), SCHEMA_CRON_EXPRESSION);
+const SCHEMA_BLOCKEDBYSAMEJOB = Joi.boolean().optional();
+const SCHEMA_BLOCKEDBYSAMEJOBWAITTIME = Joi.number()
+    .integer()
+    .positive()
+    .optional()
+    .allow(null);
```

Comment:
should we add defaults here?

Reply:
default is handled in queue-service, but yeah we can add it here too